e2e/pages/strategy.spec.ts

```diff
@@ -1,14 +1,80 @@
-import { test } from '@playwright/test';
-import { screenshot } from '../utils/operators';
+import { test, expect } from '@playwright/test';
+import { navigateTo, screenshot, waitFor } from '../utils/operators';
 import { mockApi } from '../utils/mock-api';
+import { setupImposter } from '../utils/debug';
 
 test.describe('Strategies', () => {
-  test.beforeEach(({ page }) => {
-    mockApi(page);
+  test.beforeEach(async ({ page }) => {
+    await Promise.all([mockApi(page), setupImposter(page)]);
+    await mockApi(page);
   });
   test('First Strategy Page', async ({ page }) => {
-    await page.goto('/');
+    await navigateTo(page, '/');
     await page.getByTestId('first-strategy').waitFor({ state: 'visible' });
     await screenshot(page, 'first-strategy');
   });
+
+  test('Create Limit Strategy ETH->DAI', async ({ page }) => {
+    await navigateTo(page, '/');
+    await page.getByTestId('create-strategy-desktop').click();
+
+    // Select Base
+    await page.getByTestId('select-base-token').click();
+    await waitFor(page, 'modal');
+    await page.getByLabel('Select Token').fill('eth');
+    await page.getByTestId('select-token-ETH').click();
+    await page.getByTestId('modal').waitFor({ state: 'detached' });
+
+    // Select Quote
+    await page.getByTestId('select-quote-token').click();
+    await waitFor(page, 'modal');
+    await page.getByLabel('Select Token').fill('dai');
+    await page.getByTestId('select-token-DAI').click();
+    await page.getByText('Next Step').click();
+
+    // Fill Buy fields
+    const buy = page.getByTestId('buy-section');
+    await buy.getByTestId('input-limit').fill('1500');
+    await buy.getByTestId('input-budget').fill('5000');
+    await expect(buy.getByTestId('outcome-target')).toHaveText('3.33 ETH');
```

Comment:
"target" is a confusing term here - as we use in the context of trade, and in strategies we talk base/quote (and not source/target).

Reply:
It's the name used in the FullOutcome component.
On buy it's `base` and on sell it's `quote`. Is there specific semantic to describe this ?

Reply:
we go with `value`